ACE/ace/Handle_Set.inl

```diff
@@ -24,10 +24,10 @@ ACE_Handle_Set::reset ()
 }
 
 #if defined (ACE_HAS_BIG_FD_SET)
-ACE_INLINE ACE_Handle_Set &
-ACE_Handle_Set::operator = (const ACE_Handle_Set &rhs)
+ACE_INLINE
+ACE_Handle_Set::ACE_Handle_Set (const ACE_Handle_Set &rhs)
```

Comment:
This is changing the order of the copy constructor and copy assignment operator (see line 43) which makes the diff harder to read.  Is the order significant?